tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -892,32 +892,23 @@ SpirvInstruction *SpirvEmitter::loadIfGLValue(const Expr *expr,
         const auto fromType =
```

Comment:
Why don't we move `const auto exprType = expr->getType();` at the beginning of this function?
`expr->getType()` is used in 2 more places above. It would be maybe helpful for readability and performance?!

Reply:
Done